lib/kubernetes-deploy/kubernetes_resource/daemon_set.rb

```diff
@@ -39,6 +39,7 @@ def failure_message
     end
 
     def timeout_message
+      STANDARD_TIMEOUT_MESSAGE unless @pods.present?
```

Comment:
I think you meant to `return` this

Deployment, ReplicaSet and Pod have very similar issues, where they should be returning `STANDARD_TIMEOUT_MESSAGE` and instead are returning `nil` or exactly what we were doing here. Mind fixing them too?